it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -659,6 +659,135 @@ class PersistenceConnectorSpec
       Some("<CC015C><test>testxml</test></CC015C>")
     )
 
+  "POST /traders/movements/:movementId/messages" - {
+    lazy val okResultGen =
+      for {
+        messageId <- arbitrary[MessageId]
+      } yield messageId
```

Comment:
That name is misleading, if this generates a messageId it should at least be called that for when it's seen elsewhere.

Also, I'm pretty sure you could just do `val messageIdGen = arbitrary[MessageId]`

Reply:
done the changes